frontend/src/components/GoodBye/GoodbyeMessage/GoodbyeMessage.tsx

```diff
@@ -17,7 +18,7 @@ export type GoodByeMessageProps = {
  * @returns {ReactElement} The GoodByeMessage component.
  */
 const GoodByeMessage = ({ header, message, roomName }: GoodByeMessageProps): ReactElement => {
-  const isSmallViewport = useIsSmallViewport();
+  const isSmallViewport = useMediaQuery(`(max-width:${SMALL_VIEWPORT}px)`);
```

Comment:
In this PR, we're using ```useMediaQuery(`(max-width:${SMALL_VIEWPORT}px)`)``` frequently. Does it make sense to refactor `useIsSmallViewport` as a wrapper for the MUI hook?